examples/framesjs-starter/app/page.tsx

```diff
@@ -10,12 +10,21 @@ import {
 } from "frames.js/next/server";
 import Link from "next/link";
 import { generateImage } from "./generate-image";
+import { HubHttpUrlOptions } from "frames.js";
 
 type State = {
   active: string;
   total_button_presses: number;
 };
 
+// WARN: This is a mock hub for development purposes only that does not verify signatures
```

Comment:
jsdoc style comments pls

Reply:
starts with `/**`

Reply:
I think this should have a different name, and I'd prefer it if didn't take 5 lines of space at the top of the starter as its making things more complex - can you import this from debug folder or something?